### PR TITLE
deposit naira verify deposits and withdrawals

### PR DESCRIPTION
**Pull request by manuelbiolatiri**

### CODE REVIEW

File: dist/index.d.ts [hunk 1]
```diff
@@ -125,5 +125,25 @@ export declare class ObiexClient {
         maximumDecimalPlaces: number;
     }>;
     getOrCreateWallet(currencyCode: string): Promise<Wallet>;
+    /**
+     *
+     * @param payload BankDepositRequest
+     * @returns
+     */
+    depositNaira({ merchantCode, amount }: BankDepositRequest): Promise<any>;
```
Comment (dozieogbo) [inline dist/index.d.ts:133]: Can we call this `requestNairaBankAccount` instead?

Comment (dozieogbo) [inline dist/index.d.ts]: Can we call this `requestNairaDepositBankAccount` instead?

File: dist/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Network, Options, BankAccountPayout, CryptoAccountPayout, Wallet } from "./types";
+import { Network, Options, BankAccountPayout, CryptoAccountPayout, Wallet, FiatMerchant, Banks, BankDepositRequest } from "./types";
```
Comment (dozieogbo) [inline dist/index.d.ts:1]: This should be formatted per line. Not sure why it isn't

File: dist/index.d.ts [hunk 3]
```diff
@@ -125,5 +125,25 @@ export declare class ObiexClient {
...
+    /**
+     *
+     * @param reference string
+     * @returns
+     */
+    verifyNairaWithdrawal(reference: string): Promise<any>;
 }
 export { ServerError } from "./errors/server";
+export { TransactionCategory } from "./enums/TransactionCategory";
+export { Currency, Network, Options, Quote, Response, TradePair, BankAccountPayout, CryptoAccountPayout, Wallet, FiatMerchant, Banks, BankDepositRequest, } from "./types";
```
Comment (dozieogbo) [inline dist/index.d.ts:149]: This should also be formatted per line.

**Review by dozieogbo — CHANGES_REQUESTED** [review]
Looking good. Few comments.

File: dist/index.d.ts (hunk 3 above)
Comment (manuelbiolatiri) [reply dist/index.d.ts:149]: hmmm ok I will see if I format files in dist folder

Comment (dozieogbo) [reply dist/index.d.ts:149]: Oh. It's dist. Lol. Don't worry. You can resolve this.

File: dist/index.d.ts (hunk 2 above)
Comment (dozieogbo) [reply dist/index.d.ts:1]: Resolve this as well @manuelbiolatiri

File: dist/index.d.ts (hunk 3 above)
Comment (manuelbiolatiri) [reply dist/index.d.ts:149]: ok done

**Review by dozieogbo — APPROVED** [review]
LGTM 🔥

**Pull request merged**